core/settings/staging.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
Remove unneccessary code and comments, dont repeat things that are in base.py if you are not changing them